src/scripts/keyboardNavigation.js

```diff
@@ -116,6 +131,11 @@ export function enable() {
             return;
         }
 
+        // Ignore Media Keys for non-TV platform having MediaSession API
+        if (!layoutManager.tv && isMediaKey(key) && 'mediaSession' in navigator) {
```

Comment:
I wonder which is faster, `... in ...` or a const (defined in the beginning of `enable`)? Nanoseconds (maybe a little more on old hardware or TVs), but still.